pyproject.toml

```diff
@@ -4,14 +4,18 @@ version = "0.1.0"
 description = "Translates W3D xml projects for Unity"
 authors = ["RobertGemmaJr <rgemma.jr@gmail.com>"]
 license = "MIT"
+readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-unityparser = "^2.1.5"
 typer = "^0.4.1"
+unityparser = "^2.1.5"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.scripts]
+w3d-translator ="w3d_translator.w3d_translator:main"
 
+[tool.poetry.dev-dependencies]
```

Comment:
I'd consider adding `black` too to get nice code formatting

Reply:
Ouu yes, will do. We love a code formatter